src/website/app/ErrorBoundary.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React, { Component } from 'react';
+import { createStyledComponent } from '../../styles';
+import { createThemedComponent } from '../../themes';
+import _Button from '../../Button';
+import Heading from './Heading';
+
+type Props = {
+  buttonLabel: string,
+  children?: React$Node,
+  errorMessage: string,
+  onReRender?: () => void
+};
+
+type State = {
+  hasError: boolean
+};
+
+const ErrorBlock = createStyledComponent('div', ({ theme }) => ({
+  backgroundColor: '#fce3e3', // color.red_10
```

Comment:
this seems non-cannonical to me. how would we recommend other consuming apps use our colors if they're not on our theme? whatever that it, i feel like we should do that here.

Reply:
Fair point. Updated.